AmplifyPlugins/DataStore/AWSDataStoreCategoryPluginIntegrationTests/DataStoreHubEventsTests.swift

```diff
@@ -62,6 +72,8 @@ class DataStoreHubEventTests: HubEventsIntegrationTestBase {
             return
         }
 
+        startAmplify()
```

Comment:
how does this work? Amplify.Hub.listen is called before `startAmplify()` which calls `Amplify.configure()`

Reply:
According to Tim, listening for events should work prior to `Amplify.configure()` being called. and `Combine` flavor of listening is better to be used because of its better deterministic property. So I will change it to `Hub.publisher.sink` here